### PR TITLE
Corections and minor modifications of the scil doc

### PR DESCRIPTION
**Pull request by ThoumyreStanislas**
I have made some changes to scil-documentation : 
- README : add the installation of sphinx-rtd-theme necessary to make the html
- add my name to the list of PhD student
- add link to install Apptainer
- creation of a section on the bash environment, as the 'export' command is never really explained in the docs
- correction of a few typos
- update of the various conferences
- updated zenodo link

### CODE REVIEW

File: source/intro_to/explore_os.rst [hunk 1]
```diff
@@ -292,3 +292,57 @@ These are examples of what can be done in bash in short one-line, but to get the
...
+.. code-block:: bash
+
+    vim ~/.bashrc
+    nano ~/.bashrc
+
+From your .bashrc you can modify your part default environment by adding lines to it. This includes loading modules, modifying environment variables, and activating Python virtual environments.
+.. code-block:: bash
+
+    # to load a module:
+    module load <module>
```
Comment (frheault) [inline source/intro_to/explore_os.rst:336]: Only works on ComputeCanada (or other big clusters)

File: source/intro_to/explore_os.rst [hunk 2]
```diff
@@ -292,3 +292,57 @@ These are examples of what can be done in bash in short one-line, but to get the
...
+When a program is invoked it is given an array of strings called the environment. This is a list of name-value pairs, of the form name=value.
+
+You can view all environment variables set on your system with the env command. The list is long, so pipe the output through more to make it easier to read:
+.. code-block:: bash
+
+    env | more
+
+Environment variables can be useful when you want to override default settings, or when you need to manage new settings.
+
+when you type a command, the only reason your computer knows how to find the application corresponding to that command is that the PATH environment variable tells it where to look.
```
Comment (frheault) [inline source/intro_to/explore_os.rst:308]: When

**Review by frheault — APPROVED** [review]
Did not test locally, but LGTM

**Comment by frheault** [comment]
@arnaudbore I dont have merge right for this repo

**Comment by ThoumyreStanislas** [comment]
@arnaudbore @frheault is it possible to merge this branch because I can't submit a new PR because otherwise it will pull the same changes for minor modifications.

**Pull request merged**